@navikt/core/css/darkside/accordion.darkside.css

```diff
@@ -1,86 +1,59 @@
-.navds-accordion {
-  --__ac-accordion-header-shadow-color: var(--ac-accordion-header-border, var(--a-border-divider));
-}
-
-/*************************
- * Header *
- *************************/
+/* ---------------------------- Accordion header ---------------------------- */
 .navds-accordion__header {
-  --__ac-accordion-header-shadow: inset 2px 0 0 0 var(--a-transparent), inset -2px 0 0 0 var(--a-transparent),
-    inset 0 2px 0 0 var(--__ac-accordion-header-shadow-color);
-
   width: 100%;
   display: flex;
   justify-content: flex-start;
   align-items: flex-start;
-  gap: var(--a-spacing-2);
-  padding: var(--a-spacing-3);
-  margin: 0;
+  gap: var(--ax-spacing-3);
+  padding: var(--ax-spacing-3);
   text-align: left;
-  background: var(--ac-accordion-header-bg, var(--a-surface-transparent));
   cursor: pointer;
   border: none;
   position: relative;
```

Comment:
you might want to add some sticky (and non alpha bg colors) to mirror the nav.no behaviour when you can't see the top of the accordion header anymore? 

![image](https://github.com/user-attachments/assets/f97d7c86-fa83-4031-be1d-3290b2d96f05)


```
  position: sticky;
  top: 0;
  background-color: white;
  z-index: 1;
```

Reply:
Think this feature might need to be implemented by the teams themselves, because of the possible edgecases for where "top" is. If you dig down into how they handle it, they have a custom css-variable for "top" that changes based on decorator height

Reply:
I think the top being 0 should work out as long as parent has position relative no? it seemed to work out great in my testing 🤔 The header is always inside an accordion item wrapper, so that can be the top: 0 position?